### PR TITLE
Makes integration tests run and fixes some of the awsv2 bugs

### PR DESCRIPTION
**Pull request by adriancole**
We had a blind spot I think reported before by @devinsba. Our
integration tests didn't run.

### CODE REVIEW

File: brave-instrumentation/aws-java-sdk-v2-core/pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     </dependency>
     <dependency>
       <groupId>software.amazon.awssdk</groupId>
-      <artifactId>netty-nio-client</artifactId>
+      <artifactId>url-connection-client</artifactId>
```
Comment (adriancole) [inline brave-instrumentation/aws-java-sdk-v2-core/pom.xml]: later, we'll probably want to resurrect this, without deleting the sync side. I noticed that the code paths are different and so there may be bugs on one side (async) and not the other (sync)

File: brave-instrumentation/aws-java-sdk-v2-core/src/test/java/brave/instrumentation/awsv2/ITTracingExecutionInterceptor.java [hunk 2]
```diff
@@ -18,68 +18,89 @@
...
         .region(Region.US_EAST_1)
         .overrideConfiguration(configuration)
         .endpointOverride(URI.create("http://127.0.0.1:" + i))
         .build();
   }
 
-  @Override protected void closeClient(DynamoDbAsyncClient dynamoDbAsyncClient) throws Exception {
-    dynamoDbAsyncClient.close();
+  // For some reason, the first request will fail. This primes the connection until we know why
+  SdkHttpClient primeHttpClient() {
```
Comment (adriancole) [inline brave-instrumentation/aws-java-sdk-v2-core/src/test/java/brave/instrumentation/awsv2/ITTracingExecutionInterceptor.java]: holy moly this was such a pain!

File: brave-instrumentation/aws-java-sdk-v2-core/src/test/java/brave/instrumentation/awsv2/ITTracingExecutionInterceptor.java [hunk 3]
```diff
@@ -124,68 +145,25 @@
     });
   }
 
-  /*
-   * Tests overriden due to the multi-span implementation of the client, application and client
-   */
-
-  @Override public void usesParentFromInvocationTime() throws Exception {
```
Comment (adriancole) [inline brave-instrumentation/aws-java-sdk-v2-core/src/test/java/brave/instrumentation/awsv2/ITTracingExecutionInterceptor.java]: this is an async test, so if we have to redo it, we'll need to do this too in another class.

**Comment by devinsba** [comment]
Fixes #126

**Comment by devinsba** [comment]
I wondered why I hadn't seen that flaky SQS test in a while..

**Comment by adriancole** [comment]
hehe

**Comment by adriancole** [comment]
finally green. bye circleci.. hello again travis

**Pull request merged**